app/javascript/dashboard/routes/dashboard/helpcenter/pages/categories/NameEmojiInput.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <div
+    class="flex items-center relative"
+    @mouseover="showRemoveEmojiButton = true"
+    @mouseleave="showRemoveEmojiButton = false"
```

Comment:
You could use `group-hover` from tailwind to handle this.

ref: https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-parent-state

Just add `group` class to the parent, and `hidden group-hover:block` to the icon